static/app/views/dashboards/widgetCard/chart.tsx

```diff
@@ -608,7 +608,7 @@ class WidgetCardChart extends Component<WidgetCardChartProps> {
   }
 }
 
-const getBucketSize = (series: Series[] | undefined) => {
+export const getBucketSize = (series: Series[] | undefined) => {
```

Comment:
importing this will drag in too much stuff, i'd move it to another file or copy it out